projects/steering_can/inc/steering_can.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+// Module to sets up all the interrupts for the GPIO pins to raise events
+// in the event queue when triggered
+// Requires GPIO,Interrupts,Event Queue,Soft-timer, CAN

```

Comment:
Complete your sentence: i.e.
Requires GPIO, Interrupts, Event Queues, and CAN to be initialized.
Why does this module require soft-timers to be initialized? I don't understand that. 
Again, remember to take CAN initialization out of your `steering_can_init`. Your module shouldn't initialize can. It should require the user to initialize can before using your module.